packages/web-platform/web-mainthread-apis/src/elementAPI/style/styleFunctions.ts

```diff
@@ -68,7 +68,13 @@ export function createStyleFunctions(
   ): void {
     let dashName: string | undefined;
     if (typeof key === 'number') {
-      dashName = queryCSSProperty(key).dashName;
+      const queryResult = queryCSSProperty(key);
+      dashName = queryResult.dashName;
+      if (queryResult.isX) {
+        console.error(
+          `[lynx-web] css property with index: ${key} is not supported. Please check your usage.`,
```

Comment:
Can we log the real property name instead of the number index? I don't think it would help to have an error message like: "[lynx-web] css property with index: 143 is not supported. Please check your usage."